2020-02-Spring/3_trees/README.md

```diff
@@ -116,21 +136,60 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
+### 2. Deepest Leaves Sum
 
-Source: TODO :bug:
-
-#### Naive/Simple Solution
-
-TODO :bug:
 
 #### Optimal Solution
 
-TODO :bug:
+We need the sum of the leaves at the very very last level of the tree. To do this, we 
```

Comment:
```suggestion

<details>
<summary>Click to see solution</summary>

We need the sum of the leaves at the very very last level of the tree. To do this, we 

```
This will create a nice little drop-down to hide the entire solution to prevent peeking.